nautobot_chatops/forms.py

```diff
@@ -69,41 +86,26 @@ class ChatOpsAccountLinkFilterForm(NautobotFilterForm):
     platform = forms.MultipleChoiceField(choices=PlatformChoices, required=False, widget=StaticSelect2Multiple())
 
 
-class CommandLogFilterForm(BootstrapMixin, forms.ModelForm):
+class CommandLogFilterForm(NautobotFilterForm):
     """Form for filtering Command Logs."""
 
+    platform = forms.ChoiceField(choices=PlatformChoices.CHOICES, required=False, widget=StaticSelect2Multiple())
     command = forms.CharField(required=False)
     subcommand = forms.CharField(required=False)
-
-    class Meta:
-        """Metaclass attributes of AccessGrantFilterForm."""
-
-        model = CommandLog
-
-        fields = [
-            "platform",
-            "command",
-            "subcommand",
-            "status",
-            "details",
-        ]
+    status = forms.ChoiceField(choices=BLANK_CHOICE + CommandStatusChoices.CHOICES, required=False)
```

Comment:
```suggestion
    status = forms.ChoiceField(choices=add_blank_choice(CommandStatusChoices.CHOICES), required=False)
```

need to import `add_blank_choice` from `nautobot.apps.forms`